flat.ignore.sol

```diff
@@ -0,0 +1,1459 @@
+
```

Comment:
hmm, what is this file??

Reply:
removed. It was a fragment from the sol-proxy, which flattens the contract before stripping out the variables